controllers/fenceagentsremediation_controller.go

```diff
@@ -129,6 +132,13 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		return emptyResult, nil
 	}
 
+	// Check NHC timeout annotation
+	if isTimedOutByNHC(far) {
```

Comment:
I think this can go before the if block above?

Reply:
Yes, it can